src/gps/dateTime.c

```diff
@@ -133,3 +133,118 @@ tiny_millis_t getUptime() {
 int getUptimeAsInt() {
    return (int) getUptime();
 }
+
+void getDateTimeFromEpochMillis(DateTime *dateTime, millis_t millis) {
```

Comment:
Why `void`?  Why not DateTime?  What is the benefit of making the caller allocate the space?